static/tsconfig.json

```diff
@@ -5,7 +5,7 @@
         "name": "typescript-tslint-plugin",
       }
     ],
-    "lib": ["es2019"],
+    "lib": ["es2021"],
```

Comment:
curious what the reason for this change is?

Reply:
`str.replaceAll()` requires upgrade to es2021. I use it in `parseTriplesApiResponse()` to replace single quotes with double quotes. 

If we don't like this change, another option is to do the replaceAll in the server, but splitting the post-processing between the server and client is less clean to me.

Reply:
upgrading lgtm, though see my earlier comment about moving the processing..

Reply:
this isn't required anymore, since we moved to server-side string replace.